app/views/layouts/_bugsnag_js.html.haml

```diff
@@ -2,4 +2,9 @@
 - if bugsnag_js_key.present?
   %script{src: "//d2wy8f7a9ursnm.cloudfront.net/v6/bugsnag.min.js"}
   :javascript
-    window.bugsnagClient = bugsnag("#{bugsnag_js_key}");
+    window.bugsnagClient = bugsnag({
+      apiKey: "#{bugsnag_js_key}",
+      beforeSend: function (report) {
+        report.app.releaseStage = "#{Rails.env}"
```

Comment:
we're working on exactly the same and did it differently :point_right: https://gitlab.com/coopdevs/somconnexio-tryton-webforms/-/merge_requests/62/diffs#diff-content-0efd982935b3d8d39d3b16d697f9967854d4c678

Reply:
we can check after this gets merged

Reply:
that's private

Reply:
I don't have permission to see it...

Reply:
:sob:

Reply:
In that other project this was enough:

```js
var bugsnagClient = bugsnag({
  apiKey: "{{ bugsnag_info.apiKey }}",
  releaseStage: "{{ bugsnag_info.releaseStage }}",
  appType: "{{ bugsnag_info.appType }}",
});
```

Note there was no need for `breforeSend`.